speechmatics/cli_parser.py

```diff
@@ -283,6 +283,24 @@ def get_arg_parser():
         help="Perform sentiment analysis on the transcript",
     )
 
+    # Parent parser for batch topic-detection argument
+    batch_topic_detection_parser = argparse.ArgumentParser(add_help=False)
+    batch_topic_detection_parser.add_argument(
```

Comment:
I don't know much about the API, but I'm guessing it works with sensible defaults even if you don't provide a custom list of topics?

Reply:
yes, we do have a default list of 10 topics baked into the feature, which will be used when user does not pass in a list of topics